### ISSUE TITLE
Audio/Video playback overlaps on already running video when YouTube tab is pinned

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#3817]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
Audio/Video playback overlaps on already running video when YouTube tab is pinned

**Expected behavior:**
Should continue playing the video and should not start audio/video playback from beginning.

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 x64

- Brave Version:
0.12.0 

- Steps to reproduce:
    1. Open any video on YouTube and let it play for few seconds
    2. Pin the YouTube tab while the video is running.
    3. The video/audio starts playback again from beginning. You can hear overlap of the audio which is already running and the new playback

- Screenshot if needed:

- Any related issues:

**Comment by luixxiul** [comment]
Is it a bug? Pinned tabs are loaded every time when a new window is opened, so it seems to be similar to #2227

**Comment by srirambv** [comment]
Yes pinned tabs loading when a new window is opened is fine. But it shouldn't reload the pages on already opened windows

**Comment by alexwykoff** [comment]
+1 from support : https://linkbubble.zendesk.com/agent/tickets/6310
